src/jit/codegencommon.cpp

```diff
@@ -10846,13 +10659,13 @@ void CodeGen::genSetScopeInfo()
 // Notes:
 //    Called for every scope info piece to record by the main genSetScopeInfo()
 
-void CodeGen::genSetScopeInfo(unsigned            which,
-                              UNATIVE_OFFSET      startOffs,
-                              UNATIVE_OFFSET      length,
-                              unsigned            varNum,
-                              unsigned            LVnum,
-                              bool                avail,
-                              Compiler::siVarLoc& varLoc)
+void CodeGen::genSetScopeInfo(unsigned       which,
```

Comment:
The header for that function is not filled, it would be great if you can add description for `varLoc` at least.

Reply:
I could tell you what is exactly for scopeL->scLVnum. But I have added the others.